shazib/sprint1/cdk.out/asset.0b905165ac7e424400e9f53fb11635ee2f93325f1da977adcc737a3609c3a16c/latency_handler.py

```diff
@@ -0,0 +1,35 @@
+import constants as CONSTANTS
+import urllib.request
+import time
+
+def handler(event, context):
+    print("inside latency lambda handler")
+    url = CONSTANTS.URL
+    return dict(
+            {
+                'latency': get_latency(url),
+                'availability': get_availability(url)
+            }
+        )
+
+def get_availability(url: str) -> int:
+    return 1 if urllib.request.urlopen(url).getcode() == 200 else 0
+
+def get_latency(url: str) -> float:
```

Comment:
In this function, it would be better idea to summarize the comments as doc string at the start of function, instead of having comments at every separate line. Similarly use doc string for other function too.